.github/DISCUSSION_TEMPLATE/technical-specification.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Technical Specification
+about: High level architecture for new systems/features
+title: 'Tech spec: '
+labels: technical-specification-review
+assignees: ''
+
+---
+
+## Guidelines
+
+A technical specification exists to guide implementation and to clear any ambiguity around what needs to be done in order to ship a project. The level of detail should be enough so that people won't be wondering _how_ something will be done from a high level, but it shouldn't be so much that the tech spec is basically doing the entire project.
+
+When writing technical documents, err on the side of brevity. Say just enough to get your point across.
+
+When writing tech specs, err on the side of decreasing scope instead of increasing scope if faced with the two choices.
+
+Not every section requires a response. Just leave `N/A` under a section if it's not applicable.
+
+**Delete the text above before submitting**
```

Comment:
Perhaps this should read "**Delete this and the text above before submitting.**"...